src/lib/evil/evil_sys_time.h

```diff
@@ -0,0 +1,42 @@
+#ifndef __EVIL_SYS_TIME_H__
+#define __EVIL_SYS_TIME_H__
+
+#include <evil_private.h>
+
+#include <stdint.h>
+#include <time.h>
+
+#include <winsock2.h> // struct timeval
```

Comment:
Nope, not here :) And the `evil_private.h` goes just in .c files (and Eina.h).

Not because I prefer this way, but because Evil standards. If you need these for some type definition that comes in this header, then include this on `evil_private.h`.